---
layout: post
title: "OsNOA1"
description: ""
category: genes
tags: [root, seedling, chloroplast, growth, seed, salt tolerance, salt, temperature, seed germination, vegetative, salinity]
---

* **Information**  
    + Symbol: OsNOA1  
    + MSU: [LOC_Os02g01440](http://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os02g01440)  
    + RAPdb: [Os02g0104700](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os02g0104700)  

* **Publication**  
    + [Expression of a rice gene OsNOA1 re-establishes nitric oxide synthesis and stress-related gene expression for salt tolerance in Arabidopsis nitric oxide-associated 1 mutant Atnoa1](http://www.ncbi.nlm.nih.gov/pubmed?term=Expression of a rice gene OsNOA1 re-establishes nitric oxide synthesis and stress-related gene expression for salt tolerance in Arabidopsis nitric oxide-associated 1 mutant Atnoa1%5BTitle%5D), 2009, Environmental and Experimental Botany.
    + [OsNOA1/RIF1 is a functional homolog of AtNOA1/RIF1: implication for a highly conserved plant cGTPase essential for chloroplast function](http://www.ncbi.nlm.nih.gov/pubmed?term=OsNOA1/RIF1 is a functional homolog of AtNOA1/RIF1: implication for a highly conserved plant cGTPase essential for chloroplast function%5BTitle%5D), 2010, New Phytol.
    + [NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice%5BTitle%5D), 2011, PLoS One.
    + [OsNOA1 functions in a threshold-dependent manner to regulate chloroplast proteins in rice at lower temperatures.](http://www.ncbi.nlm.nih.gov/pubmed?term=OsNOA1 functions in a threshold-dependent manner to regulate chloroplast proteins in rice at lower temperatures.%5BTitle%5D), 2018, BMC Plant Biol.

* **Genbank accession number**  
    + [AK288268](http://www.ncbi.nlm.nih.gov/nuccore/AK288268)

* **Key message**  
    + OsNOA1 transgenic expression rescued Atnoa1 in seedling development and vegetative growth under normal conditions, enhanced the salt tolerance of Atnoa1 in seed germination, seedling root growth and chlorophyll synthesis, and reduced Na+/K+ ratio in Atnoa1; NO relative content assay implicates that NO synthesis was re-established via OsNOA1 expression in Atnoa1; Northern blot and Semi-Q RT-PCR assays demonstrate that salt tolerance-related gene expression was re-established as well via OsNOA1 expression in Atnoa1
    + On the other hand, de-etiolated OsNOA1-silenced seedlings maintained similar levels of chlorophyll and Rubisco as WT, even after being shifted to 15 degrees C for various times
    + The OsNOA1/RIF1-EYFP fusion protein was targeted to chloroplasts in transgenic Arabidopsis plants
    + *Quantitative iTRAQ-LC-MS/MS proteomics investigations revealed proteome changes in the rice mutant consistent with the expected functional role of OsNOA1/RIF1 in chloroplast translation
    + OsNOA1/RIF1 is a functional homolog of AtNOA1/RIF1: implication for a highly conserved plant cGTPase essential for chloroplast function
    + *Rice (Oryza sativa) OsNOA1/RIF1 RNAi mutant seedlings were chlorotic with reduced pigment contents and lower photosystem II (PSII) efficiency
    + Our data indicate that the re-establishment of NO synthesis and salt tolerance-related gene expression by OsNOA1 expression may account for the restoration of Atnoa1 in terms of developmental and salt tolerance phenotypes
    + All the above results point to a notion that OsNOA1 may play similar roles as AtNOA1, and NO involvement in salt tolerance may be ascribed to its regulation of salt tolerance-related gene expression
    + Expression of a rice gene OsNOA1 re-establishes nitric oxide synthesis and stress-related gene expression for salt tolerance in Arabidopsis nitric oxide-associated 1 mutant Atnoa1
    + Etiolated OsNOA1-silenced plants accumulated chlorophyll and Rubisco to normal levels only at 30 degrees C, and lost this ability at low temperature
    + Overall, our results suggest OsNOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis, Rubisco formation and plastid development in rice
    + We employed the knockout mutant for AtNOA1, Atnoa1 that is sensitive to salinity, as a tool to evaluate the functions of a rice homologous gene, OsNOA1
    + OsNOA1 functions in a threshold-dependent manner to regulate chloroplast proteins in rice at lower temperatures.
    + Additionally, our data showed 90% of the suppressed proteins in both types of transgenic plants at lower temperatures were located in the chloroplast, suggesting a primary effect of OsNOA1 on chloroplast proteins
    + Transcript analyses, along with in vitro pull-down experiments further demonstrated OsNOA1 is associated with the function of chloroplast ribosomes
    + Our results suggest OsNOA1 functions in a threshold-dependent manner for regulation of chloroplast proteins at lower temperatures, which may be mediated by interactions between OsNOA1 and chloroplast ribosomes

* **Connection**  
    + __OsNOA1__, __OsPORA__, [NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice%5BTitle%5D), Further expression analyses identified several candidate genes, including OsPorA (NADPH: protochlorophyllide oxidoreductase A), OsrbcL (Rubisco large subunit), OsRALyase (Ribosomal RNA apurinic site specific lyase) and OsPuf4 (RNA-binding protein of the Puf family), which may be involved in OsNOA1-regulated chlorophyll biosynthesis and Rubisco formation
    + __OsNOA1__, __OsRALyase__, [NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice%5BTitle%5D), Further expression analyses identified several candidate genes, including OsPorA (NADPH: protochlorophyllide oxidoreductase A), OsrbcL (Rubisco large subunit), OsRALyase (Ribosomal RNA apurinic site specific lyase) and OsPuf4 (RNA-binding protein of the Puf family), which may be involved in OsNOA1-regulated chlorophyll biosynthesis and Rubisco formation
    + __OsNOA1__, __OsPuf4__, [NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=NOA1 functions in a temperature-dependent manner to regulate chlorophyll biosynthesis and Rubisco formation in rice%5BTitle%5D), Further expression analyses identified several candidate genes, including OsPorA (NADPH: protochlorophyllide oxidoreductase A), OsrbcL (Rubisco large subunit), OsRALyase (Ribosomal RNA apurinic site specific lyase) and OsPuf4 (RNA-binding protein of the Puf family), which may be involved in OsNOA1-regulated chlorophyll biosynthesis and Rubisco formation

[//]: # * **Key figures**  


