---
layout: post
title: "OsGI"
description: ""
category: genes
tags: [grain, dwarf, reproductive, sheath, vegetative, gibberellin, flower, heading date, yield, leaf, grain yield, tolerance, stress, stomata, stress response, sucrose]
---

* **Information**  
    + Symbol: OsGI  
    + MSU: [LOC_Os01g08700](http://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os01g08700)  
    + RAPdb: [Os01g0182600](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os01g0182600)  

* **Publication**  
    + [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), 2011, N Biotechnol.
    + [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), 2009, Planta.
    + [Os-GIGANTEA confers robust diurnal rhythms on the global transcriptome of rice in the field](http://www.ncbi.nlm.nih.gov/pubmed?term=Os-GIGANTEA confers robust diurnal rhythms on the global transcriptome of rice in the field%5BTitle%5D), 2011, Plant Cell.
    + [Circadian regulation of rice Oryza sativa L. CONSTANS-like gene transcripts](http://www.ncbi.nlm.nih.gov/pubmed?term=Circadian regulation of rice Oryza sativa L. CONSTANS-like gene transcripts%5BTitle%5D), 2004, Mol Cells.
    + [Adaptation of photoperiodic control pathways produces short-day flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Adaptation of photoperiodic control pathways produces short-day flowering in rice%5BTitle%5D), 2003, Nature.
    + [OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice%5BTitle%5D), 2009, Plant Cell Environ.
    + [Physiological significance of the plant circadian clock in natural field conditions](http://www.ncbi.nlm.nih.gov/pubmed?term=Physiological significance of the plant circadian clock in natural field conditions%5BTitle%5D), 2012, Plant Cell Environ.
    + [Functional analyses of the flowering time gene OsMADS50, the putative SUPPRESSOR OF OVEREXPRESSION OF CO 1/AGAMOUS-LIKE 20 SOC1/AGL20 ortholog in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional analyses of the flowering time gene OsMADS50, the putative SUPPRESSOR OF OVEREXPRESSION OF CO 1/AGAMOUS-LIKE 20 SOC1/AGL20 ortholog in rice%5BTitle%5D), 2004, Plant J.
    + [OsELF3 is involved in circadian clock regulation for promoting flowering under long-day conditions in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsELF3 is involved in circadian clock regulation for promoting flowering under long-day conditions in rice%5BTitle%5D), 2013, Mol Plant.
    + [A pair of floral regulators sets critical day length for Hd3a florigen expression in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=A pair of floral regulators sets critical day length for Hd3a florigen expression in rice%5BTitle%5D), 2010, Nat Genet.
    + [Isolation of Rice Genes Possibly Involved in the Photoperiodic Control of Flowering by a Fluorescent Differential Display Method](http://www.ncbi.nlm.nih.gov/pubmed?term=Isolation of Rice Genes Possibly Involved in the Photoperiodic Control of Flowering by a Fluorescent Differential Display Method%5BTitle%5D), 2002, Plant and Cell Physiology.
    + [Identification of dynamin as an interactor of rice GIGANTEA by tandem affinity purification TAP](http://www.ncbi.nlm.nih.gov/pubmed?term=Identification of dynamin as an interactor of rice GIGANTEA by tandem affinity purification TAP%5BTitle%5D), 2008, Plant Cell Physiol.
    + [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), 2007, Plant Physiol.
    + [Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3](http://www.ncbi.nlm.nih.gov/pubmed?term=Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3%5BTitle%5D), 2014, Plant Physiol.
    + [A study of phytohormone biosynthetic gene expression using a circadian clock-related mutant in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=A study of phytohormone biosynthetic gene expression using a circadian clock-related mutant in rice%5BTitle%5D), 2011, Plant Signal Behav.
    + [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), 2016, Front Plant Sci.
    + [Nitrogen Mediates Flowering Time and Nitrogen Use Efficiency via Floral Regulators in Rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Nitrogen Mediates Flowering Time and Nitrogen Use Efficiency via Floral Regulators in Rice%5BTitle%5D), 2020, Curr Biol.
    + [Clock component OsPRR73 positively regulates rice salt tolerance by modulating OsHKT2;1-mediated sodium homeostasis](http://www.ncbi.nlm.nih.gov/pubmed?term=Clock component OsPRR73 positively regulates rice salt tolerance by modulating OsHKT2;1-mediated sodium homeostasis%5BTitle%5D), 2020, EMBO J.
    + [Transcriptional and post-transcriptional regulation of heading date in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Transcriptional and post-transcriptional regulation of heading date in rice%5BTitle%5D), 2020, New Phytol.
    + [OsRRM, an RNA-Binding Protein, Modulates Sugar Transport in Rice  Oryza sativa L.](http://www.ncbi.nlm.nih.gov/pubmed?term=OsRRM, an RNA-Binding Protein, Modulates Sugar Transport in Rice  Oryza sativa L.%5BTitle%5D), 2020, Front Plant Sci.
    + [The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice%5BTitle%5D), 2020, Plant Cell Environ.
    + [OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions](http://www.ncbi.nlm.nih.gov/pubmed?term=OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions%5BTitle%5D), 2021, Rice (N Y) ..
    + [A clock regulatory module is required for salt tolerance and control of heading date in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=A clock regulatory module is required for salt tolerance and control of heading date in rice%5BTitle%5D), 2021, Plant Cell Environ.

* **Genbank accession number**  
    + [AJ133787](http://www.ncbi.nlm.nih.gov/nuccore/AJ133787)
    + [AP003047](http://www.ncbi.nlm.nih.gov/nuccore/AP003047)
    + [AJ133787](http://www.ncbi.nlm.nih.gov/nuccore/AJ133787)
    + [AP003047](http://www.ncbi.nlm.nih.gov/nuccore/AP003047)

* **Key message**  
    + Time-course metabolome analyses of leaves revealed no trends of change in primary metabolites in osgi plants, and net photosynthetic rates and grain yields were not affected
    + Consistently, osgi-1 plants showed semi-dwarf phenotype with reduced internode and leaf sheath elongation
    + The S3574 and C60910 genes were expressed to similar extents during the vegetative and reproductive phases, like OsGI
    + Here we found that mRNA levels of a few major genes encoding GA2-oxidase which can inactivate bioactive gibberellins (GAs) were remarkably increased in osgi-1 plants
    + Overexpression of OsGI, an orthologue of the Arabidopsis GIGANTEA (GI) gene in transgenic rice, caused late flowering under both SD and LD conditions
    + Hd3a expression is induced by Ehd1 (Early heading date 1) expression when blue light coincides with the morning phase set by OsGIGANTEA(OsGI)-dependent circadian clocks
    + In rice, OsGI, Hd1 and Hd3a were identified as orthologs of GI, CO and FT, respectively, and are also important regulators of flowering
    + Further investigations showed that the expression of OsGI and Ghd7 was up-regulated in the oself3 mutant, indicating that OsELF3 acts as a negative regulator upstream of OsGI and Ghd7 in the flowering-time control under long-day conditions
    + Transcript levels of OsGI, phytochrome genes, and Early heading date3 (Ehd3), which function upstream of Ghd7, were unchanged in the mutant
    + Recently, an extensive time-course transcriptome analysis of rice (Oryza sativa) leaves in natural field conditions revealed that OsGIGANTEA, the sole rice ortholog of the Arabidopsis GIGANTEA gene, governs the robust diurnal rhythm of rice leaf transcriptomes even under natural diurnal conditions; rice Osgi mutants exhibited severely defective transcriptome rhythms under strong diurnal changes in environmental cues
    + Flowering genes downstream of OsPRR1 such as OsGI and Hd1 were down regulated in the A654 mutant
    + Together, these results suggest that overexpression of OsLF might repress expression of OsGI and Hd1 by competing with OsPRR1 in interacting with OsPIL13 and OsPIL15 and thus induce late flowering
    + Transcript levels of three flowering regulators-Ehd1, OsMADS14, and Hd3a-were decreased in these mutants, whereas those of OsGI and Hd1 were unchanged
    + To further investigate the relationship with other flowering promoters, we generated transgenic plants in which expression of Ehd1 or OsGI was suppressed
    + In summary, OsMADS51 is a novel flowering promoter that transmits a SD promotion signal from OsGI to Ehd1
    + The expression patterns of the rice homolog of GI, OsGI, were similar to those of the Arabidopsis GI, suggesting the conservation of some mechanisms for the photoperiodic regulation of flowering between these two species
    + Results showed that mutation of OsGI conferred tolerance to osmotic stress generated by polyethylene glycol (PEG), increased proline and sucrose contents, and accelerated stomata movement
    + In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice

* **Connection**  
    + __Hd1__, __OsGI__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Flowering genes downstream of OsPRR1 such as OsGI and Hd1 were down regulated in the A654 mutant
    + __Hd1__, __OsGI__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Together, these results suggest that overexpression of OsLF might repress expression of OsGI and Hd1 by competing with OsPRR1 in interacting with OsPIL13 and OsPIL15 and thus induce late flowering
    + __OsGI__, __OsPIL13~OsPIL1__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Together, these results suggest that overexpression of OsLF might repress expression of OsGI and Hd1 by competing with OsPRR1 in interacting with OsPIL13 and OsPIL15 and thus induce late flowering
    + __OsGI__, __OsPRR1__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Flowering genes downstream of OsPRR1 such as OsGI and Hd1 were down regulated in the A654 mutant
    + __OsGI__, __OsPRR1__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Together, these results suggest that overexpression of OsLF might repress expression of OsGI and Hd1 by competing with OsPRR1 in interacting with OsPIL13 and OsPIL15 and thus induce late flowering
    + __OsGI__, __OsLF__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Together, these results suggest that overexpression of OsLF might repress expression of OsGI and Hd1 by competing with OsPRR1 in interacting with OsPIL13 and OsPIL15 and thus induce late flowering
    + __OsGI__, __OsPIL15__, [An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15](http://www.ncbi.nlm.nih.gov/pubmed?term=An atypical HLH protein OsLF in rice regulates flowering time and interacts with OsPIL13 and OsPIL15%5BTitle%5D), Together, these results suggest that overexpression of OsLF might repress expression of OsGI and Hd1 by competing with OsPRR1 in interacting with OsPIL13 and OsPIL15 and thus induce late flowering
    + __OsGI__, __OsMADS14__, [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), In transgenic lines overexpressing OsDof12, the transcription levels of Hd3a and OsMADS14 were up-regulated under LD conditions but not SD conditions, whereas the expression of Hd1, OsMADS51, Ehd1 and OsGI did not change under LD and SD conditions
    + __Hd1__, __OsGI__, [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), In transgenic lines overexpressing OsDof12, the transcription levels of Hd3a and OsMADS14 were up-regulated under LD conditions but not SD conditions, whereas the expression of Hd1, OsMADS51, Ehd1 and OsGI did not change under LD and SD conditions
    + __OsDof12~OsCDF1__, __OsGI__, [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), In transgenic lines overexpressing OsDof12, the transcription levels of Hd3a and OsMADS14 were up-regulated under LD conditions but not SD conditions, whereas the expression of Hd1, OsMADS51, Ehd1 and OsGI did not change under LD and SD conditions
    + __Ehd1__, __OsGI__, [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), In transgenic lines overexpressing OsDof12, the transcription levels of Hd3a and OsMADS14 were up-regulated under LD conditions but not SD conditions, whereas the expression of Hd1, OsMADS51, Ehd1 and OsGI did not change under LD and SD conditions
    + __Hd3a__, __OsGI__, [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), In transgenic lines overexpressing OsDof12, the transcription levels of Hd3a and OsMADS14 were up-regulated under LD conditions but not SD conditions, whereas the expression of Hd1, OsMADS51, Ehd1 and OsGI did not change under LD and SD conditions
    + __OsGI__, __OsMADS51~OsMADS65__, [Functional characterization of rice OsDof12](http://www.ncbi.nlm.nih.gov/pubmed?term=Functional characterization of rice OsDof12%5BTitle%5D), In transgenic lines overexpressing OsDof12, the transcription levels of Hd3a and OsMADS14 were up-regulated under LD conditions but not SD conditions, whereas the expression of Hd1, OsMADS51, Ehd1 and OsGI did not change under LD and SD conditions
    + __Ghd7__, __OsGI__, [OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice%5BTitle%5D), On the other hand, mRNA levels of OsGI, Hd1, OsId1, OsDof12, Ghd7, Hd6 and SE5 were unchanged
    + __Hd1__, __OsGI__, [OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice%5BTitle%5D), On the other hand, mRNA levels of OsGI, Hd1, OsId1, OsDof12, Ghd7, Hd6 and SE5 were unchanged
    + __OsDof12~OsCDF1__, __OsGI__, [OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice%5BTitle%5D), On the other hand, mRNA levels of OsGI, Hd1, OsId1, OsDof12, Ghd7, Hd6 and SE5 were unchanged
    + __Hd6~CK2__, __OsGI__, [OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS50 and OsMADS56 function antagonistically in regulating long day LD-dependent flowering in rice%5BTitle%5D), On the other hand, mRNA levels of OsGI, Hd1, OsId1, OsDof12, Ghd7, Hd6 and SE5 were unchanged
    + __Ehd1__, __OsGI__, [A pair of floral regulators sets critical day length for Hd3a florigen expression in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=A pair of floral regulators sets critical day length for Hd3a florigen expression in rice%5BTitle%5D), Hd3a expression is induced by Ehd1 (Early heading date 1) expression when blue light coincides with the morning phase set by OsGIGANTEA(OsGI)-dependent circadian clocks
    + __Hd3a__, __OsGI__, [A pair of floral regulators sets critical day length for Hd3a florigen expression in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=A pair of floral regulators sets critical day length for Hd3a florigen expression in rice%5BTitle%5D), Hd3a expression is induced by Ehd1 (Early heading date 1) expression when blue light coincides with the morning phase set by OsGIGANTEA(OsGI)-dependent circadian clocks
    + __Hd1__, __OsGI__, [Identification of dynamin as an interactor of rice GIGANTEA by tandem affinity purification TAP](http://www.ncbi.nlm.nih.gov/pubmed?term=Identification of dynamin as an interactor of rice GIGANTEA by tandem affinity purification TAP%5BTitle%5D), In rice, OsGI, Hd1 and Hd3a were identified as orthologs of GI, CO and FT, respectively, and are also important regulators of flowering
    + __Hd3a__, __OsGI__, [Identification of dynamin as an interactor of rice GIGANTEA by tandem affinity purification TAP](http://www.ncbi.nlm.nih.gov/pubmed?term=Identification of dynamin as an interactor of rice GIGANTEA by tandem affinity purification TAP%5BTitle%5D), In rice, OsGI, Hd1 and Hd3a were identified as orthologs of GI, CO and FT, respectively, and are also important regulators of flowering
    + __Ehd1__, __OsGI__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), Transcript levels of three flowering regulators-Ehd1, OsMADS14, and Hd3a-were decreased in these mutants, whereas those of OsGI and Hd1 were unchanged
    + __Ehd1__, __OsGI__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), In ectopic expression lines, transcript levels of Ehd1, OsMADS14, and Hd3a were increased, but those of OsGI and Hd1 remained the same
    + __Ehd1__, __OsGI__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), To further investigate the relationship with other flowering promoters, we generated transgenic plants in which expression of Ehd1 or OsGI was suppressed
    + __Ehd1__, __OsGI__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), In summary, OsMADS51 is a novel flowering promoter that transmits a SD promotion signal from OsGI to Ehd1
    + __Hd3a__, __OsGI__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), Transcript levels of three flowering regulators-Ehd1, OsMADS14, and Hd3a-were decreased in these mutants, whereas those of OsGI and Hd1 were unchanged
    + __Hd3a__, __OsGI__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), In ectopic expression lines, transcript levels of Ehd1, OsMADS14, and Hd3a were increased, but those of OsGI and Hd1 remained the same
    + __OsGI__, __OsMADS14__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), Transcript levels of three flowering regulators-Ehd1, OsMADS14, and Hd3a-were decreased in these mutants, whereas those of OsGI and Hd1 were unchanged
    + __OsGI__, __OsMADS14__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), In ectopic expression lines, transcript levels of Ehd1, OsMADS14, and Hd3a were increased, but those of OsGI and Hd1 remained the same
    + __OsGI__, __OsMADS51~OsMADS65__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), However, in OsGI antisense plants, the OsMADS51 transcript level was reduced
    + __OsGI__, __OsMADS51~OsMADS65__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), These results demonstrate that OsMADS51 functions downstream of OsGI
    + __OsGI__, __OsMADS51~OsMADS65__, [OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a](http://www.ncbi.nlm.nih.gov/pubmed?term=OsMADS51 is a short-day flowering promoter that functions upstream of Ehd1, OsMADS14, and Hd3a%5BTitle%5D), In summary, OsMADS51 is a novel flowering promoter that transmits a SD promotion signal from OsGI to Ehd1
    + __Ehd3__, __OsGI__, [Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3](http://www.ncbi.nlm.nih.gov/pubmed?term=Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3%5BTitle%5D), Transcript levels of OsGI, phytochrome genes, and Early heading date3 (Ehd3), which function upstream of Ghd7, were unchanged in the mutant
    + __Ghd7__, __OsGI__, [Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3](http://www.ncbi.nlm.nih.gov/pubmed?term=Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3%5BTitle%5D), Transcript levels of OsGI, phytochrome genes, and Early heading date3 (Ehd3), which function upstream of Ghd7, were unchanged in the mutant
    + __FKF1~OsFKF1__, __OsGI__, [Rice FLAVIN-BINDING, KELCH REPEAT, F-BOX 1 OsFKF1 promotes flowering independent of photoperiod.](http://www.ncbi.nlm.nih.gov/pubmed?term=Rice FLAVIN-BINDING, KELCH REPEAT, F-BOX 1 OsFKF1 promotes flowering independent of photoperiod.%5BTitle%5D), Like Arabidopsis FKF1, which interacts with GI and CDF1, OsFKF1 also interacts with OsGI and OsCDF1 (also termed OsDOF12)
    + __OsGI__, __PHYA~OsPhyA__, [OsPhyA modulates rice flowering time mainly through OsGI under short days and Ghd7 under long days in the absence of phytochrome B.](http://www.ncbi.nlm.nih.gov/pubmed?term=OsPhyA modulates rice flowering time mainly through OsGI under short days and Ghd7 under long days in the absence of phytochrome B.%5BTitle%5D), OsPhyA modulates rice flowering time mainly through OsGI under short days and Ghd7 under long days in the absence of phytochrome B.
    + __OsGI__, __PHYA~OsPhyA__, [OsPhyA modulates rice flowering time mainly through OsGI under short days and Ghd7 under long days in the absence of phytochrome B.](http://www.ncbi.nlm.nih.gov/pubmed?term=OsPhyA modulates rice flowering time mainly through OsGI under short days and Ghd7 under long days in the absence of phytochrome B.%5BTitle%5D), These results indicated that OsPhyA influences flowering time mainly by affecting the expression of OsGI under SD and Ghd7 under LD when phytochrome B is absent
    + __OsAP37__, __OsGI__, [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice
    + __OsGI__, __OsLEA3~OsLEA3-1__, [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice
    + __OsGI__, __OsSalT~OsMBL1__, [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice
    + __AP59~OsBIERF3~OsAP59__, __OsGI__, [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice
    + __OsDhn1~OsLip9__, __OsGI__, [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice
    + __OsDREB1E~OsDREB1-1~CR350__, __OsGI__, [Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Mutation of OsGIGANTEA Leads to Enhanced Tolerance to Polyethylene Glycol-Generated Osmotic Stress in Rice.%5BTitle%5D), In addition, qRT-PCR and microarray analysis revealed that the transcript abundance of some osmotic stress response genes, such as OsDREB1E, OsAP37, OsAP59, OsLIP9, OsLEA3, OsRAB16A, and OsSalT, was significantly higher in osgi than in WT plants, suggesting that OsGI might be a negative regulator in the osmotic stress response in rice
    + __OsGI__, __ROC4__, [GL2-type homeobox gene Roc4 in rice promotes flowering time preferentially under long days by repressing Ghd7](http://www.ncbi.nlm.nih.gov/pubmed?term=GL2-type homeobox gene Roc4 in rice promotes flowering time preferentially under long days by repressing Ghd7%5BTitle%5D), sativa GIGANTEA (OsGI) function upstream of Roc4
    + __OsGI__, __ROC4__, [GL2-type homeobox gene Roc4 in rice promotes flowering time preferentially under long days by repressing Ghd7](http://www.ncbi.nlm.nih.gov/pubmed?term=GL2-type homeobox gene Roc4 in rice promotes flowering time preferentially under long days by repressing Ghd7%5BTitle%5D), All of these findings are evidence that Roc4 is an LD-preferential flowering enhancer that functions downstream of phytochromes and OsGI, but upstream of Ghd7
    + __LTN1~OsPHO2~OsRLS1__, __OsGI__, [Molecular interaction between PHO2 and GI in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular interaction between PHO2 and GI in rice.%5BTitle%5D),  Characterization of rice Osgi and Ospho2 mutants revealed that they displayed several similar phenotypic features supporting a physiological role for this interaction
    + __LTN1~OsPHO2~OsRLS1__, __OsGI__, [Molecular interaction between PHO2 and GI in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular interaction between PHO2 and GI in rice.%5BTitle%5D),  Reduced growth, leaf tip necrosis, delayed flowering and over-accumulation of Pi in leaves compared to wild-type were shared features of Osgi and Ospho2 plants
    + __LTN1~OsPHO2~OsRLS1__, __OsGI__, [Molecular interaction between PHO2 and GI in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular interaction between PHO2 and GI in rice.%5BTitle%5D),  Pi analysis of individual leaves demonstrated that Osgi, similar to Ospho2 mutants, were impaired in Pi remobilization from old to young leaves, albeit to a lesser extent
    + __LTN1~OsPHO2~OsRLS1__, __OsGI__, [Molecular interaction between PHO2 and GI in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular interaction between PHO2 and GI in rice.%5BTitle%5D),  Transcriptome analyses revealed more than 55% of the genes differentially expressed in Osgi plants overlapped with the set of differentially expressed genes in Ospho2 plants
    + __LTN1~OsPHO2~OsRLS1__, __OsGI__, [Molecular interaction between PHO2 and GI in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Molecular interaction between PHO2 and GI in rice.%5BTitle%5D),  The interaction between OsPHO2 and OsGI links high-level regulators of Pi homeostasis and development in rice
    + __OsGI__, __OsMADS7~OsMADS45__, [Ectopic expression of OsMADS45 activates the upstream genes Hd3a and RFT1 at an early development stage causing early flowering in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Ectopic expression of OsMADS45 activates the upstream genes Hd3a and RFT1 at an early development stage causing early flowering in rice.%5BTitle%5D),  OsMADS45 overexpression did not influence other floral regulator genes upstream of Hd1 and Ehd1, such as OsGI (OsGIGANTEA), Ehd2/Osld1/RID1 and OsMADS50
    + __OsGI__, __OsIDS1__, [Histone Deacetylase 701 HDT701 Induces Flowering in Rice by Modulating Expression of OsIDS1.](http://www.ncbi.nlm.nih.gov/pubmed?term=Histone Deacetylase 701 HDT701 Induces Flowering in Rice by Modulating Expression of OsIDS1.%5BTitle%5D),  Transcript levels of OsINDETERMINATE SPIKELET 1 (OsIDS1), an upstream repressor of Ehd1, were significantly increased in the mutants while those of OsGI and Hd1 were reduced
    + __Ghd7__, __OsGI__, [Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.%5BTitle%5D), Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.
    + __Ghd7__, __OsGI__, [Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.%5BTitle%5D),  We also found that co-expressing OsGI with Ghd7 causes reduced accumulation of Ghd7 protein and partially suppresses the delayed flowering phenotype in the wild type background, suggesting that phytochromes and OsGI play antagonist roles in regulating Ghd7 protein stability and flowering time
    + __Ghd7__, __OsGI__, [Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.%5BTitle%5D),  We show that OsPHYA, OsPHYB and OsGI could directly interact with Ghd7
    + __Ghd7__, __OsGI__, [Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.%5BTitle%5D),  Interestingly, OsPHYA and OsPHYB could inhibit the interaction between OsGI and Ghd7, and thus helping to stabilize Ghd7 protein
    + __Ghd7__, __OsGI__, [Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=Posttranscriptional regulation of Ghd7 protein stability by phytochrome and OsGI in photoperiodic control of flowering in rice.%5BTitle%5D),  Our results revealed a new level of Ghd7 regulation by phytochromes and OsGI in photoperiodic control of flowering in rice
    + __OsCCT11~OsPRR73__, __OsGI__, [The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice%5BTitle%5D),  In LL and DD, the oscillatory patterns of the circadian genes OsLHY, OsTOC1, OsGI and OsELF3 were varied in OsPRR73OX and osprr73 mutants
    + __OsGI__, __OsLHY~OsCCA1__, [The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice%5BTitle%5D),  In LL and DD, the oscillatory patterns of the circadian genes OsLHY, OsTOC1, OsGI and OsELF3 were varied in OsPRR73OX and osprr73 mutants
    + __Hd17~Ef7~OsELF3-1~OsELF3.1~OsELF3__, __OsGI__, [The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=The transcriptional repressor OsPRR73 links circadian clock and photoperiod pathway to control heading date in rice%5BTitle%5D),  In LL and DD, the oscillatory patterns of the circadian genes OsLHY, OsTOC1, OsGI and OsELF3 were varied in OsPRR73OX and osprr73 mutants
    + __Ghd7.1~Hd2~OsPRR37~DTH7__, __OsGI__, [OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions](http://www.ncbi.nlm.nih.gov/pubmed?term=OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions%5BTitle%5D),  Further investigation showed that the alternative function of OsPRR37 required an intact Ghd7-related regulatory pathway involving not only its upstream regulators OsGI and PhyB but also its interacting partner Hd1
    + __Hd1__, __OsGI__, [OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions](http://www.ncbi.nlm.nih.gov/pubmed?term=OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions%5BTitle%5D),  Further investigation showed that the alternative function of OsPRR37 required an intact Ghd7-related regulatory pathway involving not only its upstream regulators OsGI and PhyB but also its interacting partner Hd1
    + __OsGI__, __PHYB~OsphyB__, [OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions](http://www.ncbi.nlm.nih.gov/pubmed?term=OsPRR37 Alternatively Promotes Heading Date Through Suppressing the Expression of Ghd7 in the Japonica Variety Zhonghua 11 under Natural Long-Day Conditions%5BTitle%5D),  Further investigation showed that the alternative function of OsPRR37 required an intact Ghd7-related regulatory pathway involving not only its upstream regulators OsGI and PhyB but also its interacting partner Hd1
    + __OsGI__, __OsLHY__, [Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice%5BTitle%5D),  Subsequent one-hybrid assays in yeast, DNA affinity purification qPCR, and electrophoretic mobility shift assays confirmed OsLHY could directly bind to the CBS element in OsGI promoter
    + __OsGI__, __OsLHY__, [Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice%5BTitle%5D),  Moreover, the critical day length (CDL) for function reversal of OsLHY in oslhy (11-12 h) was prolonged in the double mutant oslhy osgi (about 13
    + __OsGI__, __OsLHY__, [Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice%5BTitle%5D), 5 h), indicating that the CDL set by OsLHY was OsGI dependent
    + __OsGI__, __OsLHY__, [Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Dual function of clock component OsLHY sets critical day length for photoperiodic flowering in rice%5BTitle%5D),  Together, OsLHY could fine-tune the CDL by directly regulating OsGI, and Hd1 acts as the final effector of CDL downstream of OsLHY
    + __OsGI__, __OsLHY__, [mediated pathways in rice Oryza sativa L..](http://www.ncbi.nlm.nih.gov/pubmed?term=mediated pathways in rice Oryza sativa L..%5BTitle%5D),  Dual-luciferase assays showed that OsLHY repressed the transcription of OsGI, Hd1, Ghd7, Hd3a, RFT1 and OsELF3, and activated the transcription of Ehd1
    + __OsGI__, __OsLHY__, [mediated pathways in rice Oryza sativa L..](http://www.ncbi.nlm.nih.gov/pubmed?term=mediated pathways in rice Oryza sativa L..%5BTitle%5D),  Moreover, the yeast one hybrid assay and electrophoretic mobility shift assay confirmed that OsLHY directly repressed OsGI, RFT1 and OsELF3 by binding to their promoters, which is consistent with that in Arabidopsis
    + __Hd17~Ef7~OsELF3-1~OsELF3.1~OsELF3__, __OsGI__, [mediated pathways in rice Oryza sativa L..](http://www.ncbi.nlm.nih.gov/pubmed?term=mediated pathways in rice Oryza sativa L..%5BTitle%5D),  Dual-luciferase assays showed that OsLHY repressed the transcription of OsGI, Hd1, Ghd7, Hd3a, RFT1 and OsELF3, and activated the transcription of Ehd1
    + __Hd17~Ef7~OsELF3-1~OsELF3.1~OsELF3__, __OsGI__, [mediated pathways in rice Oryza sativa L..](http://www.ncbi.nlm.nih.gov/pubmed?term=mediated pathways in rice Oryza sativa L..%5BTitle%5D),  Moreover, the yeast one hybrid assay and electrophoretic mobility shift assay confirmed that OsLHY directly repressed OsGI, RFT1 and OsELF3 by binding to their promoters, which is consistent with that in Arabidopsis
    + __OsGI__, __OsHUB2~FRRP1__, [OsHUB2 inhibits function of OsTrx1 in heading date in rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=OsHUB2 inhibits function of OsTrx1 in heading date in rice.%5BTitle%5D),  The expression of Hd3a, RFT1, and Ehd1 was induced and the transcript levels of Hd1, Ghd7, OsCCA1, OsGI, OsFKF1, and OsTOC1 were reduced under long-day conditions, whereas RFT1 and Ehd1 expression was induced in oshub2 mutants under short-day conditions
    + __OsGI__, __SDG724~lvp1__, [Crosstalk between the Circadian Clock and Histone Methylation.](http://www.ncbi.nlm.nih.gov/pubmed?term=Crosstalk between the Circadian Clock and Histone Methylation.%5BTitle%5D),  Furthermore, we identified three candidate targets (OsGI, OsCCT38, and OsPRR95) of OsLHY and one candidate target (OsCRY1a) of SDG724 in the clock pathway
    + __OsGI__, __OsLHY__, [Crosstalk between the Circadian Clock and Histone Methylation.](http://www.ncbi.nlm.nih.gov/pubmed?term=Crosstalk between the Circadian Clock and Histone Methylation.%5BTitle%5D),  Furthermore, we identified three candidate targets (OsGI, OsCCT38, and OsPRR95) of OsLHY and one candidate target (OsCRY1a) of SDG724 in the clock pathway

[//]: # * **Key figures**  


