---
layout: post
title: "BAD1,OsBADH1"
description: ""
category: genes
tags: [salt, drought, salinity, temperature, salt stress, salt tolerance, salinity stress, submergence, tolerance, stress]
---

* **Information**  
    + Symbol: BAD1,OsBADH1  
    + MSU: [LOC_Os04g39020](http://rice.uga.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os04g39020)  
    + RAPdb: [Os04g0464200](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os04g0464200)  

* **Publication**  
    + [OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes](http://www.ncbi.nlm.nih.gov/pubmed?term=OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes%5BTitle%5D), 2009, FEBS Lett.
    + [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), 2012, Biochimie.
    + [Genetic manipulation of Japonica rice using the OsBADH1 gene from Indica rice to improve salinity tolerance](http://www.ncbi.nlm.nih.gov/pubmed?term=Genetic manipulation of Japonica rice using the OsBADH1 gene from Indica rice to improve salinity tolerance%5BTitle%5D), 2010, Plant Cell, Tissue and Organ Culture (PCTOC).
    + [Expression of OsBADH1 gene in Indica rice Oryza sativaL. in correlation with salt, plasmolysis, temperature and light stresses ](http://www.ncbi.nlm.nih.gov/pubmed?term=Expression of OsBADH1 gene in Indica rice Oryza sativaL. in correlation with salt, plasmolysis, temperature and light stresses %5BTitle%5D), 2011, Plant Omics.
    + [Inactivation of an aminoaldehyde dehydrogenase is responsible for fragrance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Inactivation of an aminoaldehyde dehydrogenase is responsible for fragrance in rice%5BTitle%5D), 2008, Plant Mol Biol.
    + [Expression of Indica rice OsBADH1 gene under salinity stress in transgenic tobacco](http://www.ncbi.nlm.nih.gov/pubmed?term=Expression of Indica rice OsBADH1 gene under salinity stress in transgenic tobacco%5BTitle%5D), 2009, Plant Biotechnology Reports.
    + [RNAi-directed downregulation of betaine aldehyde dehydrogenase 1 OsBADH1 results in decreased stress tolerance and increased oxidative markers without affecting glycine betaine biosynthesis in rice Oryza sativa.](http://www.ncbi.nlm.nih.gov/pubmed?term=RNAi-directed downregulation of betaine aldehyde dehydrogenase 1 OsBADH1 results in decreased stress tolerance and increased oxidative markers without affecting glycine betaine biosynthesis in rice Oryza sativa.%5BTitle%5D), 2014, Plant Mol Biol.
    + [Resequencing Reveals Different Domestication Rate for BADH1 and BADH2 in Rice Oryza sativa.](http://www.ncbi.nlm.nih.gov/pubmed?term=Resequencing Reveals Different Domestication Rate for BADH1 and BADH2 in Rice Oryza sativa.%5BTitle%5D), 2015, PLoS One.

* **Genbank accession number**  
    + [BAA21098](http://www.ncbi.nlm.nih.gov/nuccore/BAA21098)

* **Key message**  
    + Northern blot analysis revealed that salt-tolerant in each rice cultivar is correlated to the expression level of OsBADH1 mRNA
    + Moreover, these results suggest that the expression of OsBADH1 gene in response to salt stress could be magnified under high light conditions
    + Interestingly, the effect of salt stress on the expression of OsBADH1 gene was alleviated by CO2 enrichment
    + Expression of OsBADH1 gene in Indica rice (Oryza sativaL.) in correlation with salt, plasmolysis, temperature and light stresses
    + We previously found that the expression of betaine aldehyde dehydrogenase 1 gene (OsBADH1), encoding a key enzyme for glycine betaine biosynthesis pathway, showed close correlation with salt tolerance of rice
    + In this study, the expression of the OsBADH1 gene in transgenic tobacco was investigated in response to salt stress using a transgenic approach
    + The expression studies showed that OsBADH1 can be induced by a variety of environmental factors such as salinity, drought, cold, heat, light intensity and CO2 concentration
    + The results demonstrated that the OsBADH1 mRNA expression was up-regulated by salinity, drought, cold and high light intensity but down-regulated by CO2 enrichment and heat stress
    + The primary response of OsBADH1 gene expression was induced within 24 h after salinity, cold or drought stress treatment
    + Expression of Indica rice OsBADH1 gene under salinity stress in transgenic tobacco
    + Japonica rice (salt-sensitive) was genetically engineered to enhance salt tolerance by introducing the OsBADH1 gene from Indica rice (salt-tolerant), which is a GB accumulator
    + Genetic manipulation of Japonica rice using the OsBADH1 gene from Indica rice to improve salinity tolerance
    + The accumulation of OsBADH1 mRNA decreases following submergence treatment, but quickly recovers after re-aeration
    + Transgenic rice lines downregulating OsBADH1 exhibited remarkably reduced tolerance to NaCl, drought and cold stresses
    + In this study, we demonstrated a pivotal role of OsBADH1 in stress tolerance without altering GB biosynthesis capacity, using the RNA interference (RNAi) technique

* **Connection**  
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes](http://www.ncbi.nlm.nih.gov/pubmed?term=OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes%5BTitle%5D), We found that OsBADH1 catalyzes the oxidation of acetaldehyde efficiently, while the activity of OsBADH2 is extremely low
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), Fragrance rice (Oryza sativa) contains two isoforms of BADH, named OsBADH1 and OsBADH2
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), OsBADH1 is implicated in acetaldehyde oxidation in rice plant peroxisomes, while the non-functional OsBADH2 is believed to be involved in the accumulation of 2-acetyl-1-pyrroline, the major compound of aroma in fragrance rice
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), Consistent with our previous study, kinetics data indicated that the enzymes catalyze the oxidation of GAB-ald more efficiently than Bet-ald and the OsBADH1 W172F and OsBADH2 W170F mutants displayed a higher catalytic efficiency towards GAB-ald
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), The amino acid residues, E262, L263, C296 and W461 of OsBADH1 and E260, L261, C294 and W459 of OsBADH2 located within 5 A of the OsBADH active site mainly interacted with GAB-ald forming strong hydrogen bonds in both OsBADH isoforms
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), Residues W163, N164, Q294, C296 and F397 of OsBADH1-Bet-ald and Y163, M167, W170, E260, S295 and C453 of OsBADH2-Bet-ald formed the main interaction sites while E260 showed an interaction energy of -14
    + __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), Unconserved A290 in OsBADH1 and W288 in OsBADH2 appeared to be important for substrate recognition similar to that observed in PsAMADHs

[//]: # * **Key figures**  


