---
layout: post
title: "OsBADH2,fgr"
description: ""
category: genes
tags: [root, Gene]
---

## Information
__Symbol__: OsBADH2,fgr  
__MSU__: [LOC_Os08g32870](http://rice.plantbiology.msu.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os08g32870)  
__RAPdb__: [Os08g0424500](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os08g0424500)  

## Publication
1. [Aroma in rice: genetic analysis of a quantitative trait](http://www.ncbi.nlm.nih.gov/pubmed?term=Aroma in rice: genetic analysis of a quantitative trait%5BTitle%5D), 1996, Theor Appl Genet.
2. [OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes](http://www.ncbi.nlm.nih.gov/pubmed?term=OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes%5BTitle%5D), 2009, FEBS Lett.
3. [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), 2012, Biochimie.
4. [Haplotype variation at Badh2, the gene determining fragrance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Haplotype variation at Badh2, the gene determining fragrance in rice%5BTitle%5D), 2012, Genomics.
5. [RFLP tagging of a gene for aroma in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=RFLP tagging of a gene for aroma in rice%5BTitle%5D), 1992, Theor Appl Genet.
6. [Characterization of the major fragance gene from an aromatic japonica rice and analysis of its diversity in Asian cultivated rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Characterization of the major fragance gene from an aromatic japonica rice and analysis of its diversity in Asian cultivated rice%5BTitle%5D), 2008, Theor Appl Genet.
7. [RNAi-directed downregulation of OsBADH2 results in aroma 2-acetyl-1-pyrroline production in rice Oryza sativa L.](http://www.ncbi.nlm.nih.gov/pubmed?term=RNAi-directed downregulation of OsBADH2 results in aroma 2-acetyl-1-pyrroline production in rice Oryza sativa L.%5BTitle%5D), 2008, BMC Plant Biol.
8. [Purification, crystallization and preliminary X-ray analysis of recombinant betaine aldehyde dehydrogenase 2 OsBADH2), a protein involved in jasmine aroma, from Thai fragrant rice Oryza sativa L.](http://www.ncbi.nlm.nih.gov/pubmed?term=Purification, crystallization and preliminary X-ray analysis of recombinant betaine aldehyde dehydrogenase 2 OsBADH2), a protein involved in jasmine aroma, from Thai fragrant rice Oryza sativa L.%5BTitle%5D), 2011, Acta Crystallogr Sect F Struct Biol Cryst Commun.
9. [The gene for fragrance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=The gene for fragrance in rice%5BTitle%5D), 2005, Plant Biotechnol J.
10. [Badh2, encoding betaine aldehyde dehydrogenase, inhibits the biosynthesis of 2-acetyl-1-pyrroline, a major component in rice fragrance](http://www.ncbi.nlm.nih.gov/pubmed?term=Badh2, encoding betaine aldehyde dehydrogenase, inhibits the biosynthesis of 2-acetyl-1-pyrroline, a major component in rice fragrance%5BTitle%5D), 2008, Plant Cell.
11. [The origin and evolution of fragrance in rice Oryza sativa L.](http://www.ncbi.nlm.nih.gov/pubmed?term=The origin and evolution of fragrance in rice Oryza sativa L.%5BTitle%5D), 2009, Proc Natl Acad Sci U S A.
12. [Biochemical and enzymatic study of rice BADH wild-type and mutants: an insight into fragrance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Biochemical and enzymatic study of rice BADH wild-type and mutants: an insight into fragrance in rice%5BTitle%5D), 2011, Protein J.
13. [Inactivation of an aminoaldehyde dehydrogenase is responsible for fragrance in rice](http://www.ncbi.nlm.nih.gov/pubmed?term=Inactivation of an aminoaldehyde dehydrogenase is responsible for fragrance in rice%5BTitle%5D), 2008, Plant Mol Biol.
14. [Creation of fragrant rice by targeted knockout of the OsBADH2 gene using TALEN technology.](http://www.ncbi.nlm.nih.gov/pubmed?term=Creation of fragrant rice by targeted knockout of the OsBADH2 gene using TALEN technology.%5BTitle%5D), 2015, Plant Biotechnol J.
15. [Resequencing Reveals Different Domestication Rate for BADH1 and BADH2 in Rice Oryza sativa.](http://www.ncbi.nlm.nih.gov/pubmed?term=Resequencing Reveals Different Domestication Rate for BADH1 and BADH2 in Rice Oryza sativa.%5BTitle%5D), 2015, PLoS One.

## Genbank accession number
[AK071221](http://www.ncbi.nlm.nih.gov/nuccore/AK071221), [EU770319](http://www.ncbi.nlm.nih.gov/nuccore/EU770319), [EU770320](http://www.ncbi.nlm.nih.gov/nuccore/EU770320), [EU770321](http://www.ncbi.nlm.nih.gov/nuccore/EU770321)  

## Expression information

## Key message
1. __root__, [RNAi-directed downregulation of OsBADH2 results in aroma 2-acetyl-1-pyrroline production in rice Oryza sativa L.](http://www.ncbi.nlm.nih.gov/pubmed?term=RNAi-directed downregulation of OsBADH2 results in aroma 2-acetyl-1-pyrroline production in rice Oryza sativa L.%5BTitle%5D),  RESULTS: The results showed that multiple mutations identical to fgr allele occur in the 13 fragrant rice accessions across China; OsBADH2 is expressed constitutively, with less expression abundance in mature roots; the disrupted OsBADH2 by RNA interference leads to significantly increased 2-acetyl-1-pyrroline production

## Connection
1. __BAD1~OsBADH1__, __OsBADH2~fgr__, [OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes](http://www.ncbi.nlm.nih.gov/pubmed?term=OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes%5BTitle%5D), 8 gene homologs OsBADH1 and OsBADH2)  
2. __BAD1~OsBADH1__, __OsBADH2~fgr__, [OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes](http://www.ncbi.nlm.nih.gov/pubmed?term=OsBADH1 is possibly involved in acetaldehyde oxidation in rice plant peroxisomes%5BTitle%5D),  We found that OsBADH1 catalyzes the oxidation of acetaldehyde efficiently, while the activity of OsBADH2 is extremely low
3. __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D), Fragrance rice Oryza sativa contains two isoforms of BADH, named OsBADH1 and OsBADH2
4. __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D),  OsBADH1 is implicated in acetaldehyde oxidation in rice plant peroxisomes, while the non-functional OsBADH2 is believed to be involved in the accumulation of 2-acetyl-1-pyrroline, the major compound of aroma in fragrance rice
5. __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D),  Consistent with our previous study, kinetics data indicated that the enzymes catalyze the oxidation of GAB-ald more efficiently than Bet-ald and the OsBADH1 W172F and OsBADH2 W170F mutants displayed a higher catalytic efficiency towards GAB-ald
6. __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D),  The amino acid residues, E262, L263, C296 and W461 of OsBADH1 and E260, L261, C294 and W459 of OsBADH2 located within 5 A of the OsBADH active site mainly interacted with GAB-ald forming strong hydrogen bonds in both OsBADH isoforms
7. __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D),  Residues W163, N164, Q294, C296 and F397 of OsBADH1-Bet-ald and Y163, M167, W170, E260, S295 and C453 of OsBADH2-Bet-ald formed the main interaction sites while E260 showed an interaction energy of -14
8. __BAD1~OsBADH1__, __OsBADH2~fgr__, [Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies](http://www.ncbi.nlm.nih.gov/pubmed?term=Dissecting substrate specificity of two rice BADH isoforms: Enzyme kinetics, docking and molecular dynamics simulation studies%5BTitle%5D),  Unconserved A290 in OsBADH1 and W288 in OsBADH2 appeared to be important for substrate recognition similar to that observed in PsAMADHs

## Key figures
<img src="http://ricencode.github.io/images/OsBADH2~fgr.pheno.png" alt="phenotype"  style="width: 600px;"/>

<img src="http://ricencode.github.io/images/OsBADH2~fgr.exp.png" alt="expression"  style="width: 600px;"/>


