---
layout: post
title: "NRR,CRCT"
description: ""
category: genes
tags: [root, root architecture, seedling, growth, architecture, leaf, tiller, sheath, starch, vegetative, photosynthesis, phloem, tiller angle, sucrose, plant growth]
---

## Information
__Symbol__: NRR,CRCT  
__MSU__: [LOC_Os05g51690](http://rice.plantbiology.msu.edu/cgi-bin/ORF_infopage.cgi?orf=LOC_Os05g51690)  
__RAPdb__: [Os05g0595300](http://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=Os05g0595300)  

## Publication
1. [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D), 2012, Mol Plant.
2. [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D), 2015, Plant Physiol.

## Genbank accession number
[AK059614](http://www.ncbi.nlm.nih.gov/nuccore/AK059614)  

## Expression information
__Expression__: The expression of CRCT showed diurnal oscillation peaked at the end of light period and was also increased by sugars such as glucose and sucrose |  Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression |  The expressions of several genes related to starch synthesis such as ADP-glucose pyrophospholylase and α-glucan phospholylase were significantly changed in transgenic lines and positively correlated with the expression levels of CRCT |  Given these observations, we suggest that CRCT is a positive regulator of starch accumulation in vegetative tissues, regulating coordinated expression of starch synthesis genes in response to the levels of photoassimilates  
__OverExpression__: Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression  
__RNAi__: Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression  

## Key message
1. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  We report here the expression and biological functions of a previously uncharacterized rice gene that we have named NRR nutrition response and root growth)  
2. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Expression of NRR in rice seedling roots was significantly influenced by deficiency of macronutrients
3. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Knock-down of expression of NRRa or NRRb by RNA interference resulted in enhanced rice root growth
4. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  By contrast, overexpression of NRRa in rice exhibited significantly retarded root growth
5. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  These results revealed that both NRRa and NRRb played negative regulatory roles in rice root growth
6. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Our findings suggest that NRRa and NRRb, acting as the key components, modulate the rice root architecture with the availability of macronutrients
7. __root__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D), A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth
8. __root architecture__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Our findings suggest that NRRa and NRRb, acting as the key components, modulate the rice root architecture with the availability of macronutrients
9. __seedling__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Expression of NRR in rice seedling roots was significantly influenced by deficiency of macronutrients
10. __growth__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  We report here the expression and biological functions of a previously uncharacterized rice gene that we have named NRR nutrition response and root growth)  
11. __growth__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Knock-down of expression of NRRa or NRRb by RNA interference resulted in enhanced rice root growth
12. __growth__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  By contrast, overexpression of NRRa in rice exhibited significantly retarded root growth
13. __growth__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  These results revealed that both NRRa and NRRb played negative regulatory roles in rice root growth
14. __growth__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D), A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth
15. __architecture__, [A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth](http://www.ncbi.nlm.nih.gov/pubmed?term=A novel rice gene, NRR responds to macronutrient deficiency and regulates root growth%5BTitle%5D),  Our findings suggest that NRRa and NRRb, acting as the key components, modulate the rice root architecture with the availability of macronutrients
16. __leaf__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Promoter GUS analysis showed that CRCT was highly expressed in the phloem of various tissues such as leaf blade and leaf sheath
17. __growth__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression
18. __tiller__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression
19. __sheath__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Promoter GUS analysis showed that CRCT was highly expressed in the phloem of various tissues such as leaf blade and leaf sheath
20. __starch__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D), CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.
21. __starch__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  The expressions of several genes related to starch synthesis such as ADP-glucose pyrophospholylase and α-glucan phospholylase were significantly changed in transgenic lines and positively correlated with the expression levels of CRCT
22. __starch__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Given these observations, we suggest that CRCT is a positive regulator of starch accumulation in vegetative tissues, regulating coordinated expression of starch synthesis genes in response to the levels of photoassimilates
23. __vegetative__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D), CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.
24. __vegetative__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Given these observations, we suggest that CRCT is a positive regulator of starch accumulation in vegetative tissues, regulating coordinated expression of starch synthesis genes in response to the levels of photoassimilates
25. __photosynthesis__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression
26. __phloem__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Promoter GUS analysis showed that CRCT was highly expressed in the phloem of various tissues such as leaf blade and leaf sheath
27. __tiller angle__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression
28. __sucrose__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  The expression of CRCT showed diurnal oscillation peaked at the end of light period and was also increased by sugars such as glucose and sucrose
29. __plant growth__, [CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.](http://www.ncbi.nlm.nih.gov/pubmed?term=CO2 Responsive CCT protein, CRCT Is a Positive Regulator of Starch Synthesis in Vegetative Organs of Rice.%5BTitle%5D),  Overexpression or RNAi knockdown of CRCT had no appreciable effect on plant growth and photosynthesis except that tiller angle was significantly increased by the overexpression

## Connection
1. __NRR~CRCT__, __OsNPR1~NH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  NRR disrupts NH1 function, when over-expressed
2. __NRR~CRCT__, __OsNPR1~NH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  RESULTS: We have established a rice transient protoplast assay to demonstrate that NH1 is a transcriptional co-activator and that NRR represses NH1-mediated activation
3. __NRR~CRCT__, __OsNPR1~NH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  NRR, RH1, RH2, and RH3 share sequence similarity in a region beyond the previously identified NPR1-interacting domain
4. __NRR~CRCT__, __OsNPR1~NH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  Mutation W66A/F70A also greatly reduces the ability of NRR to repress NH1-mediated activation
5. __NRR~CRCT__, __OsNPR1~NH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  Our results demonstrate that the ability of NRR and its homologues to repress NH1-mediated transcriptional activation is tightly correlated with their ability to bind to NH1
6. __NRR~CRCT__, __OsNPR1~NH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D), A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation
7. __NRR~CRCT__, __RH3__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  We identified three NRR homologues RH1, RH2, and RH3)  
8. __NRR~CRCT__, __RH3__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  NRR, RH1, RH2, and RH3 share sequence similarity in a region beyond the previously identified NPR1-interacting domain
9. __NRR~CRCT__, __RH3__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  RH2 carries a deviation amino acids AV in this region as compared to consensus sequences amino acids ED among NRR, RH1, and RH3
10. __NRR~CRCT__, __RH2__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  We identified three NRR homologues RH1, RH2, and RH3)  
11. __NRR~CRCT__, __RH2__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  NRR, RH1, RH2, and RH3 share sequence similarity in a region beyond the previously identified NPR1-interacting domain
12. __NRR~CRCT__, __RH2__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  RH2 carries a deviation amino acids AV in this region as compared to consensus sequences amino acids ED among NRR, RH1, and RH3
13. __NRR~CRCT__, __RH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  We identified three NRR homologues RH1, RH2, and RH3)  
14. __NRR~CRCT__, __RH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  NRR, RH1, RH2, and RH3 share sequence similarity in a region beyond the previously identified NPR1-interacting domain
15. __NRR~CRCT__, __RH1__, [A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation](http://www.ncbi.nlm.nih.gov/pubmed?term=A rice transient assay system identifies a novel domain in NRR required for interaction with NH1/OsNPR1 and inhibition of NH1-mediated transcriptional activation%5BTitle%5D),  RH2 carries a deviation amino acids AV in this region as compared to consensus sequences amino acids ED among NRR, RH1, and RH3
16. __NRR~CRCT__, __OsNPR1~NH1__, [Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1](http://www.ncbi.nlm.nih.gov/pubmed?term=Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1%5BTitle%5D),  NRR interacts with NPR1 in the NPR1-interacting domain NI25 consisting of 25 amino acids
17. __NRR~CRCT__, __OsNPR1~NH1__, [Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1](http://www.ncbi.nlm.nih.gov/pubmed?term=Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1%5BTitle%5D),  NRR also interacts with NH1; however, NI25 was not sufficient for a strong interaction, indicating a difference between the rice and the Arabidopsis proteins
18. __NRR~CRCT__, __OsNPR1~NH1__, [Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1](http://www.ncbi.nlm.nih.gov/pubmed?term=Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1%5BTitle%5D),  The fact that NRR compromises Xa21-mediated resistance indicates cross-talk or overlap between NH1- and Xa21-mediated pathways
19. __NRR~CRCT__, __OsNPR1~NH1__, [Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1](http://www.ncbi.nlm.nih.gov/pubmed?term=Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1%5BTitle%5D), Rice NRR, a negative regulator of disease resistance, interacts with Arabidopsis NPR1 and rice NH1

## Key figures
<img src="http://ricencode.github.io/images/NRR-growth.pheno.png" alt="phenotype"  style="width: 600px;"/>

<img src="http://ricencode.github.io/images/NRR-growth.exp.png" alt="expression"  style="width: 600px;"/>


